From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Julian Brown <julian.brown@cpanel.net>
Date: Fri, 4 Oct 2024 11:18:45 -0500
Subject: [PATCH 01/11] EA4 OBS ready

---
 build/php.m4       | 1 -
 sapi/fpm/config.m4 | 2 --
 2 files changed, 3 deletions(-)

diff --git a/build/php.m4 b/build/php.m4
index 1e0b282..0c7e551 100644
--- a/build/php.m4
+++ b/build/php.m4
@@ -1902,7 +1902,6 @@ dnl
 dnl Common setup macro for libxml.
 dnl
 AC_DEFUN([PHP_SETUP_LIBXML], [
-  PKG_CHECK_MODULES([LIBXML], [libxml-2.0 >= 2.9.4])
   PHP_EVAL_INCLINE([$LIBXML_CFLAGS])
   PHP_EVAL_LIBLINE([$LIBXML_LIBS], [$1])
   $2
diff --git a/sapi/fpm/config.m4 b/sapi/fpm/config.m4
index 4d4952e..495309e 100644
--- a/sapi/fpm/config.m4
+++ b/sapi/fpm/config.m4
@@ -399,8 +399,6 @@ if test "$PHP_FPM" != "no"; then
     [no])
 
   AS_VAR_IF([PHP_FPM_SYSTEMD], [no], [php_fpm_systemd=simple], [
-    PKG_CHECK_MODULES([SYSTEMD], [libsystemd >= 209])
-
     AC_DEFINE([HAVE_SYSTEMD], [1],
       [Define to 1 if FPM has systemd integration.])
     PHP_FPM_SD_FILES="fpm/fpm_systemd.c"
