From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Brian Mendoza <brian.mendoza@cpanel.net>
Date: Mon, 10 Oct 2022 14:46:34 +0000
Subject: [PATCH 05/12] Ensure that php.d is not scanned when PHPRC is set

---
 main/php_ini.c | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/main/php_ini.c b/main/php_ini.c
index 82420fd..1a28bab 100644
--- a/main/php_ini.c
+++ b/main/php_ini.c
@@ -628,7 +628,15 @@ int php_init_config(void)
 	php_ini_scanned_path_len = strlen(php_ini_scanned_path);
 
 	/* Scan and parse any .ini files found in scan path if path not empty. */
-	if (!sapi_module.php_ini_ignore && php_ini_scanned_path_len) {
+
+    /* But only do this if PHPRC is undefined.  This way the ini files defined
+     * in the php.d directory do not override the user's custom settings.
+     *
+     * We don't leverage .user.ini due to EA3 backwards-compatibility.
+     *
+     * Case: EA-4422
+     */
+    if (!getenv("PHPRC") && !sapi_module.php_ini_ignore && php_ini_scanned_path_len) {
 		struct dirent **namelist;
 		zend_stat_t sb = {0};
 		char ini_file[MAXPATHLEN];
